src/main/java/com/volunteer/enterprise/service/VolunteerServiceStub.java

```diff
@@ -11,34 +11,30 @@
 @Service
 public class VolunteerServiceStub implements IVolunteerService {
     @Autowired
-    private IVolunteerService volunteerService;
+    private IVolunteerDAO volunteerDAO;
 
     public VolunteerServiceStub(){
 
     }
 
     @Override
     public Volunteer save(Volunteer volunteer) throws Exception {
-        return null;
+        return volunteerDAO.save(volunteer);
     }
 
     @Override
     public List<Volunteer> fetchAll() {
-        return null;
-    }
-
-    @Override
-    public Volunteer fetch(int id) {
-        return null;
+        return volunteerDAO.fetchAll();
     }
 
     @Override
     public void delete(int id) {
-
+        volunteerDAO.delete(id);
     }
 
     @Override
     public Volunteer fetchById(int id) {
-        return null;
+        Volunteer foundVolunteer = volunteerDAO.fetch(id);
+        return foundVolunteer;
```

Comment:
Properly implemented `VolunteerServiceStub` so its not just returning nulls.
Removed redundant methods from `VolunteerService` & `IVolunteerService`.